record-and-playback/core/lib/recordandplayback/generators/events.rb

```
@@ -732,6 +732,38 @@ def self.anonymous_user_map(events, moderators: false)
       map
     end
 
+    def self.get_chat_reactions(events)
+      reactions = {}
+      events.xpath('/recording/event').each do |event|
+        case [event[:module], event[:eventname]]
+        when %w[CHAT SendPublicChatMessageReactionRecordEvent]
+          reaction_emoji = event.at_xpath('./reactionEmoji')&.content
+          message_id = event.at_xpath('./messageId')&.content
+          if reactions[message_id] != nil  then
```

rubocop

[convention] Prefer `!reactions[message_id].nil?` over `reactions[message_id] != nil`.

(Style/NonNilCheck)